kubeflow-pipelines/docling-vlm/docling_convert_components.py

```
@@ -264,6 +260,33 @@ def docling_convert(
         )
 
     if remote_model_enabled:
+        if not os.path.exists(remote_model_secret_mount_path):
+            raise ValueError(f"Secret for remote model should be mounted in {remote_model_secret_mount_path}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)